pandas/core/strings.py

```diff
@@ -1842,23 +1842,45 @@ def extractall(self, pat, flags=0):
         return str_extractall(self._orig, pat, flags=flags)
 
     _shared_docs['find'] = ("""
+    Return the lowest index of the substring.
```

Comment:
Can you change 'lowest' to `` %(side)s `` ? (so it gets correctly substituted)

Reply:
Done!.